### PR TITLE
chore: activate pa11y and publish report

### PR DESCRIPTION
**Pull request by plagoa**

### CODE REVIEW

File: .github/workflows/tests-pa11y.yml [hunk 1]
```diff
@@ -35,4 +35,31 @@ jobs:
...
+    name: Publish pa11y Report
+    needs: run-tests
+    if: always()
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Deploy Artifacts
```
Comment (zettca) [inline .github/workflows/tests-pa11y.yml:56]: I don't think that we need a "whole job" for this, we can `if: always()` a step

```suggestion
      - name: Deploy Artifacts
        if: always()
```

File: .github/workflows/tests-pa11y.yml [hunk 2]
```diff
@@ -35,4 +35,31 @@ jobs:
...
           "npx http-server dist --port 6006 --silent" \
-          "npx wait-on tcp:6006 && npm run test:pa11y"
\ No newline at end of file
+          "npx wait-on tcp:6006 && npm run test:pa11y" || true
+
+      - name: Archive pa11y report
+        uses: actions/upload-artifact@v3
+        with:
+          name: pa11y-report
+          path: ${{ github.workspace }}/pa11y-ci-report
```
Comment (zettca) [inline .github/workflows/tests-pa11y.yml:44]: We're storing quite a few things now. We should probably define a shorter [retention-period](https://github.com/actions/upload-artifact#retention-period) here.
We can look into this later

**Comment by zettca** [comment]
Report is looking great BTW 👌 🚀

**Review by zettca — APPROVED** [review]
:shipit:

**Review by MEsteves22 — APPROVED** [review]

**Review by francisco-guilherme — APPROVED** [review]

**Pull request merged**